File: app/src/main/java/com/yenvth/soilDetectionApp/history/HistoryView.java
package com.yenvth.soilDetectionApp.history;


import com.yenvth.soilDetectionApp.models.HistoryModel;

import java.util.ArrayList;

public interface HistoryView {
    void onGetListHistorySuccess(ArrayList<HistoryModel> historyModels);

    void onUpdateHistorySuccess();

    void onDeleteHistorySuccess();
}

File: app/src/main/java/com/yenvth/soilDetectionApp/history/HistoryView.kt
package com.yenvth.soilDetectionApp.history

import com.yenvth.soilDetectionApp.models.HistoryModel

interface HistoryView {
    fun onGetListHistorySuccess(historyModels: List<HistoryModel>)
    fun onDeleteHistorySuccess()
    fun onDeleteAllHistoriesSuccess()
}